lldb/source/Plugins/Platform/WebAssembly/PlatformWasmProperties.td

```diff
@@ -0,0 +1,21 @@
+include "../../../../include/lldb/Core/PropertiesBase.td"
+
+let Definition = "platformwasm" in {
+  def RuntimePath
+      : Property<"runtime-path", "FileSpec">,
+        Global,
+        DefaultStringValue<"">,
+        Desc<"Path to the WebAssembly runtime binary. If the path does not "
+             "contain a directory separator, the filename is looked up in the "
+             "PATH environment variable. If empty, the filename is derived "
+             "from the architecture setting.">;
```

Comment:
> If empty, the filename is derived from the architecture setting.

As far as I can tell it is not for wasm. For qemu it's easy, qemu + architecture usually works.

I assume that here you can't do that unless there's some accepted `wasm` global symlink that folks use.